cocos/ui/components/widget-component.ts

```diff
@@ -331,6 +375,10 @@ export class WidgetComponent extends Component {
     }
 
     /**
+     * @en
+     * Whether the stretched vertically, when enable the left and right alignment will be stretched vertically,
```

Comment:
Whether the stretched vertically, when enable the left and right alignment will be stretched vertically,
-->
Whether to stretch vertically, when enabled the left and right alignment will be stretched vertically,